tests/regression/lp-1721518/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Check that interfaces are listed after reboot the machine
+
+details: |
+    This test checks if the interfaces are listed after the machine
+    is rebooted. The test is also checking the interfaces after a 
+    snap is installed and removed.
+
+execute: |
+    # install test-snapd-tools and reboot
+    if [ "$SPREAD_REBOOT" = 0 ]; then
+        snap install test-snapd-tools
```

Comment:
Can you please move `CONNECTED_PATTERN=":core-support +core:core-support-plug"` above and then add the same test after the snap is installed but before we reboot? `snap interfaces | MATCH "$CONNECTED_PATTERN"`